packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet.tsx

```diff
@@ -227,7 +182,18 @@ export class AtomicSmartSnippet implements InitializableComponent {
           {this.renderQuestion()}
           {this.renderContent()}
           <footer part="footer">
-            {this.renderSource()}
+            {this.smartSnippetState.source && (
+              <atomic-smart-snippet-source
+                source={this.smartSnippetState.source}
+                selectSource={() => this.smartSnippet.selectSource()}
+                beginDelayedSelectSource={() =>
+                  this.smartSnippet.beginDelayedSelectSource()
+                }
+                cancelPendingSelectSource={() =>
+                  this.smartSnippet.cancelPendingSelectSource()
+                }
```

Comment:
If you decide to use events, these will be automatically renamed. Otherwise, looks good!
```suggestion
                onSelectSource={() => this.smartSnippet.selectSource()}
                onBeginDelayedSelectSource={() =>
                  this.smartSnippet.beginDelayedSelectSource()
                }
                onCancelPendingSelectSource={() =>
                  this.smartSnippet.cancelPendingSelectSource()
                }
```